src/ctia/stores/es/crud.clj

```diff
@@ -691,6 +699,7 @@ It returns the documents with full hits meta data including the real index in wh
           :topn make-topn
           :cardinality make-cardinality
           :histogram make-histogram
+          :avg make-average
```

Comment:
can you add a test case here, please?
https://github.com/threatgrid/ctia/blob/c00cacb4a79f4b634d687aee5b3d5fd5e991f607/test/ctia/stores/es/crud_test.clj#L539